Gruntfile.coffee

```diff
@@ -183,8 +176,10 @@ module.exports = (grunt) ->
 		# Metadata.
 		pkg: @file.readJSON "package.json"
 		themeDist: "dist/<%= pkg.name %>"
-		jqueryVersion: @file.readJSON "lib/jquery/bower.json"
-		jqueryOldIEVersion: @file.readJSON "lib/jquery-oldIE/bower.json"
+		jqueryVersion: grunt.file.readJSON(
```

Comment:
I did something "fancy" here in the upstream, but we could always just hardcode it like the OldIE one

Reply:
This works without hardcoding so it makes it easier to maintain